config/schedule.rb

```diff
@@ -0,0 +1,23 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+# every 2.hours do
+#   command "/usr/bin/some_great_command"
+#   runner "MyModel.some_method"
+#   rake "some:great:rake:task"
+# end
+#
+# every 4.days do
+#   runner "AnotherModel.prune_old_records"
+# end
+
+# Learn more: http://github.com/javan/whenever
+every 1.minute do
+  runner "UpdateApiDataJob.perform_now"
```

Comment:
very cool logic and helpful for you specific project. I know the commented out code is from the gem but if youre not using it anymore i would delete it just to tidy up files!